recipes-multimedia/gstreamer/gstreamer1.0_1.10.4.bb

```diff
@@ -0,0 +1,18 @@
+require gstreamer1.0.inc
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=6762ed442b3822387a51c92d928ead0d \
```

Comment:
Could you please indent it with 4 spaces?

Reply:
done